frontend/src/components/Dashboard/Dashboard.js

```diff
@@ -136,7 +137,9 @@ const Dashboard = props => {
         paper: classes.drawerPaper
       }}
     >
-      <div className={classes.drawerHeader} />
+      <div className={classes.drawerHeader}>
+        <DrawerManager userName="Олег притула" tel="093-111-11-11" email="olegprutula@gmail.com" />
```

Comment:
functionality is not finalized
if the backend is not yet ready, then use mocks at the editor level with the return of an asynchronous mock response